build/generate-docs.ts

```diff
@@ -190,7 +197,9 @@ function convertPropertyToMarkdown(key: string, value: JsonObject, keyPrefix = '
     if (paintLayoutText) {
         markdown += `[${paintLayoutText}](#${paintLayoutText.toLowerCase()}) property. `;
     }
-    const valueType = typeToMakrdownLink(value.type);
+
+    const mdLink = typeToMarkdownLink(value.type);
+    const valueType = mdLink ? ` ${formatMdLink(mdLink)}` : '';
```

Comment:
this doesn't take into consideration the `null` return value where it should not create an extra space.